docs/faq/errors.md

```diff
@@ -0,0 +1,48 @@
+# Errors and Common Problems #
+
+## **Paymetheus** ##
+
+#### **1. I've just started using Paymetheus, but it seems to be stuck.** ####
+
+The first time you start Paymetheus, it will download the blockchain. this can take up
+to an hour and Paymetheus will appear to be doing nothing. This will only happen once.
+
+#### **2. Invalid passphrase for master private key.** ####
+
+This is just a long way of saying, "Incorrect password". You entered the wrong password for your
+wallet.
+
+#### **3. "Unable to purchase tickets: insufficient funds available..." but the wallet says I have enough.** ####
+
+There is a known bug in Paymetheus where immature funds as counted as available. After a ticket votes, there is a 
+256 block window where the funds are still locked. In this state, they are known as immature. When the period expires
+they will be usable again.
+
+#### **4. Paymetheus is displaying the wrong balance.** ####
+
+> These instructions are valid as of version 0.8.x and may not work with later versions.
+
+If Paymetheus displays the wrong balance, you can fix it by using the command line utility to overwrite some files.
+Some of this can be confusing if you are not familiar with the command line, but just follow the instructions line by line
+and you'll be fine. Where you see commands that `look like this`, just copy them and paste them exactly as they are into the command line.
+Don't forget to press <ENTER> after each command. If you get stuck, join our [Slack channel](https://decred.slack.com) and someone will help,
```

Comment:
Oops, Glad you picked that up. Not all of them were formatted the same either. Fixed.